products/sle/main.pm

```diff
@@ -469,7 +469,7 @@ sub load_inst_tests() {
             && sle_version_at_least('12-SP2')
             && is_server()
             && (!is_sles4sap() || is_sles4sap_standard())
-            && install_this_version())
+            && (install_this_version() || install_to_other_at_least('12-SP2')))
```

Comment:
Before sle12sp2, there is no system_role selection during installation, so this test module should only be loaded for those sle12sp2+